addons/mentions/userscript.js

```diff
@@ -0,0 +1,39 @@
+export default async function ({ addon }) {
+  if (!(await addon.auth.fetchIsLoggedIn())) {
```

Comment:
Note that `scratch-www` allows dynamic logins.

Reply:
(Logging in without having to reload the page)